homeassistant/components/nuheat/__init__.py

```diff
@@ -109,7 +109,7 @@ async def _async_update_data():
         """Fetch data from API endpoint."""
         try:
             await hass.async_add_executor_job(thermostat.get_data)
-        except Exception as err:
+        except requests.exceptions.RequestException as err:
```

Comment:
I think we're already handling this exception the same way in the coordinator helper:
https://github.com/home-assistant/core/blob/6c3362217efad87426f674e0a4b8e61bfab051d0/homeassistant/helpers/update_coordinator.py#L151-L154

Reply:
nice. I'll pull it out.